src/Base/Console.h

```diff
@@ -495,18 +495,18 @@ class BaseExport ConsoleSingleton
 {
 
 public:
-
+    static const unsigned int BufferSize = 4024;

```

Comment:
@wwmayer 

Are you ok with this refactoring?

P.S.: I do not know if you are receiving this duplicated. I have inadvertently clicked "update branch". It created a new merge commit and outdated my previous comment... :(